lib/logger/renovate-logger.ts

```diff
@@ -0,0 +1,128 @@
+import is from '@sindresorhus/is';
+import type * as bunyan from 'bunyan';
+import { once, reset as onceReset } from './once';
+import { getRemappedLevel } from './remap';
+import type { Logger } from './types';
+import { getMessage, toMeta, withSanitizer } from './utils';
+
+const loggerLevels: bunyan.LogLevelString[] = [
+  'trace',
+  'debug',
+  'info',
+  'warn',
+  'error',
+  'fatal',
+];
+
+type loggerFunction = (p1: string | Record<string, any>, p2?: string) => void;
```

Comment:
```suggestion
type LoggerFunction = (p1: string | Record<string, unknown>, p2?: string) => void;
```

Types should start with UpperCase

Reply:
i love golang too much i guess :)